### DISCUSSION TITLE
change font

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#31] [General]
Hi there

I tried to change the fonts in the template revitron/automad-revitron to google dosis font in several ways. Nothing did work for me. It seem as the system doesn't change anything when I write some code in the custom html or css section.

What is the best way to change the font?

thanks in advance.
Greg

**Comment by marcantondahmen** [comment]
Hi, you might want to check the actual CSS rules that are actually applied. Also it depends where you put the customizations.

If you place custom CSS in the shared settings, it will be used on all pages that don't have their own customization. It acts as a default. As soon as you add custom CSS to a page, the shared one is ignored.
